### PR TITLE
Seers dialogues and other fixes

### PR DESCRIPTION
**Pull request by Indio3**
- dialogues for seer, bartender (seers village), and kebab seller, small inaccuracy fixes for banker/tower guard
- added grinding chocolate bars -> dust + fixed other messages
- fixed delay on hay searching

### CODE REVIEW

File: data/src/scripts/general_use/scripts/haybales.rs2 [hunk 1]
```diff
@@ -16,7 +16,7 @@ mes("You search the hay bail...");
 
 [label,search_haybale]()
 anim(bury, 0);
-p_delay(1);
+p_delay(3);
```
Comment (ultraviolet-jordan) [inline data/src/scripts/general_use/scripts/haybales.rs2:19]: This should be 3 ticks according to this:
image

`p_delay(2);`

File: data/src/scripts/general_use/scripts/spade.rs2 [hunk 2]
```diff
@@ -28,6 +28,4 @@ if(~trail_hasclue_inv = true) {
     }
 }
 
-~trail_clue_easy_reward;
```
Comment (ultraviolet-jordan) [inline data/src/scripts/general_use/scripts/spade.rs2:31]: Is this supposed to be removed?

File: data/src/scripts/general_use/scripts/haybales.rs2 (hunk 1 above)
Comment (Indio3) [reply data/src/scripts/general_use/scripts/haybales.rs2:19]: didn't realize it was - 1, i'll change it

File: data/src/scripts/general_use/scripts/spade.rs2 (hunk 2 above)
Comment (Indio3) [reply data/src/scripts/general_use/scripts/spade.rs2:31]: I think so, figured it was probably accidentally left in from testing (it'll send an easy clue reward on any dig where you don't have a map/coord clue in inv)

Comment (FlenarnTemp) [reply data/src/scripts/general_use/scripts/spade.rs2:31]: Correct! It was used to generate up clue rewards during development, should be removed.

**Pull request merged**